internal/auth0/organization/resource_member_role.go

```diff
@@ -0,0 +1,134 @@
+package organization
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/auth0/go-auth0/management"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/auth0/terraform-provider-auth0/internal/config"
+	internalSchema "github.com/auth0/terraform-provider-auth0/internal/schema"
+)
+
+// NewMemberRoleResource will return a new auth0_organization_member_role (1:1) resource.
+func NewMemberRoleResource() *schema.Resource {
+	return &schema.Resource{
+		Description:   "This resource is used to manage the roles assigned to an organization member.",
+		CreateContext: createOrganizationMemberRole,
+		ReadContext:   readOrganizationMemberRole,
+		DeleteContext: deleteOrganizationMemberRole,
+		Importer: &schema.ResourceImporter{
+			StateContext: internalSchema.ImportResourcePairID("organization_id", "user_id"),
```

Comment:
Appears that the role ID is missing from the import process. Probably requires modification to the helper function or require bespoke logic.

Reply:
Updated!